### PR TITLE
Added Mask R-CNN

### PR DESCRIPTION
**Pull request by poornima2605**

### CODE REVIEW

File: examples/mask_rcnn/tests/pipeline_test.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+
+def test_compute_anchor_boxes_overlaps(anchors, anchor_boxes_overlaps):
+    overlaps, no_crowd_bool = anchor_boxes_overlaps
+    assert anchors.shape[0] == overlaps.shape[0] == no_crowd_bool.shape[0]
+
+
+def test_rpn_match(rpn_match):
+    rpn, anchor_iou_max = rpn_match
+    assert np.all(np.unique(rpn) == (-1, 0, 1))
```
Comment (proneetsharma) [inline examples/mask_rcnn/tests/pipeline_test.py:124]: numpy import is missing

Comment (oarriaga) [reply examples/mask_rcnn/tests/pipeline_test.py:124]: Yes. Would it be possible to add the import in the PR @proneetsharma ?

**Comment by oarriaga** [comment]
@poornima2605 do you think you could help us with the current errors? Thank you!

**Comment by poornima2605** [comment]
Closing this PR. Will Create a new PR for mask R-CNN.

**Pull request closed without merging**